extide/o.apache.tools.ant.module/build.xml

```diff
@@ -35,7 +35,7 @@
                 <path path="${src-bridge.cp}"/>
             </classpath>
         </depend>
-        <javac srcdir="src-bridge" destdir="build/bridge-classes" deprecation="${build.compiler.deprecation}" debug="${build.compiler.debug}" source="1.6" target="1.6" includeantruntime="false">
+        <javac srcdir="src-bridge" destdir="build/bridge-classes" deprecation="${build.compiler.deprecation}" debug="${build.compiler.debug}" source="1.8" target="1.8" includeantruntime="false">
```

Comment:
Consider adding `release="8"` attribute or using `nb-javac` which is supposed to do that automatically.

Reply:
8 doesn't know `--release`. So i would have to do both, switch to nb-javac and replace both attributes with the release attribute i suppose.

Could we leave this as is until we move to JDK 11+? After that everything will be easier. I am not sure if its worth to put a lot of energy into this, since its only the basic cluster - i expect a lot more changes. It doesn't look like JDK will drop support for source/target any time soon and it does currently allow NB to be built on JDK 17.

Once NB dropped 8 support it would be a fairly simple change. Remove the build support logic Matthias mentioned and switch everything to `javac.release`.

Reply:
@mbien the Ant compiler task ignores `release` on JDK 8.

Reply:
@neilcsmith-net interesting! didn't know that. 

plus: "When set and running on JDK 9+ the source and target attributes as well as the bootclasspath will be ignored. Since Ant 1.9.8."

which means we can set all three to the same value? I would be ok with this.

Reply:
@mbien yes, as far as I know. Even an argument that only release is necessary, as the only JDK that's going to ignore it will default to source and target being 8 anyway.  But being specific is probably better!

Reply:
pushed update:
 - `release` requires the new versioning scheme on modern JDKs -> so i have to use 8 instead of 1.8
 - nb-javac throws NPEs when only release is set -> added source and target there too
 - javac now uses only release on all xml files i touched for this PR
 - exception: debugger.jpda.ant can't use release without it causing compiler errors -> kept source/target
 - rebased ontop of latest master, no need to pull gradle changes anymore

Reply:
> set all three to the same value

Looks like Ant's approach is better than Maven's one (fail on JDK8 when release is used) when shielding developers from JDK differences.

> nb-javac throws NPEs when only release is set -> added source and target there too

You can fix the task in `nbbuild/antsrc`...

Reply:
> You can fix the task in nbbuild/antsrc...

yes i do want to take a look later. We shouldn't have a configuration divergence between the two javac tasks if its avoidable.

Reply:
added 1d773e0